megatron/core/models/mamba/mamba_model.py

```diff
@@ -63,6 +74,7 @@ def __init__(
         hybrid_attention_ratio: float = 0.0,
         hybrid_mlp_ratio: float = 0.0,
         hybrid_override_pattern: str = None,
+        mtp_hybrid_override_pattern: str = None,
```

Comment:
I think this should be called `mtp_layer_pattern`. But why is is not part of the overall layer pattern?